tests/modified_tait_forsterite_box.prm

```diff
@@ -0,0 +1,92 @@
+# This test ensures that the Modified Tait
```

Comment:
you'll need an output file for this test to be useful :-)

Reply:
Oops! Thanks, this also allowed me to catch a bug.